myblog/blog/api/serializers.py

```diff
@@ -7,4 +7,20 @@ class BlogPostSerializer(serializers.ModelSerializer):
     
     class Meta:
         model = BlogPost
-        fields = ('body', 'title', 'author')
\ No newline at end of file
+        fields = ('body', 'title', 'author')
+
+class CommentSerializer(serializers.ModelSerializer):
+    blogpost = serializers.SerializerMethodField()
```

Comment:
Another way to achieve the same result is using nested serializer, [more info here](https://www.django-rest-framework.org/api-guide/relations/#nested-relationships)